packages/dx-grid-core/src/plugins/grouping-panel/computeds.js

```diff
@@ -0,0 +1,4 @@
+export const ungroupedColumns = (columns, grouping) => [
```

Comment:
Why do you use `[...array]` syntax? It is redundant.

Also, I am not sure that this computed related to GroupingPanel. It seems that it should be used by TableGroupRow.